nservicebus/upgrades/support-policy.md

```diff
@@ -39,6 +39,13 @@ include: supported-versions-nservicebus
 For other NServiceBus packages, refer to [the list of supported versions of all packages](supported-versions.md).
 
 
+## Extended Support Policy
+
+Extended support is helpful when more time is needed to upgrade an existing system on an unsupported version of NServiceBus to a supported version of NServiceBus. In order to be entitled to extended support, a valid support agreement with Particular is needed. Extended support provides an extra two years at the end of the mainstream support. During this period:
+
+   - Older samples are still provided to customers.
+   - If a bug is detected and reported, Particular will triage it and provide a patch if it is deemed critical.
+
```

Comment:
Are you running your system on an unsupported version of NServiceBus? Extended Support provides you with two more years of support, beginning at the end of mainstream support, allowing you to upgrade your system at your own pace. In order to purchase extended support, a valid support agreement with Particular is needed.  

During this period:
* Older samples continue to be provided to customers
* If a bug is detected and reported, Particular will triage it and provide a patch - as long as it is deemed critical

Reply:
This suggestion won't work because it uses personal voice (the build will fail).